app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+class MoviesController < ApplicationController
+  include JSONAPI::Fetching
+  skip_before_action :authenticate_user!, only: %i[show index]
```

Comment:
Why? ;>

Reply:
I want the users to access the movies list and specific movies without the requirement to be logged in.